js/imagediff.js

```diff
@@ -343,16 +374,16 @@
     toImageData : function (object) {
       checkType(object);
       if (isImageData(object)) { return copyImageData(object); }
-      return toImageData(object);
+      return toImageData.apply(null, arguments);
     },
 
-    equal : function (a, b, tolerance) {
+    equal : function (a, b, tolerance, callback) {
       checkType(a, b);
       a = toImageData(a);
       b = toImageData(b);
       return equal(a, b, tolerance);
     },
-    diff : function (a, b) {
+    diff : function (a, b, callback) {
```

Comment:
In some cases the image is not loaded immediately.  We must wait for `image.onload` but I never finished wiring it up.